source/code/include/playfab/PlayFabBaseModel.h

```diff
@@ -97,35 +77,12 @@ namespace PlayFab
     // Utilities for [de]serializing time_t to/from json
     inline void ToJsonUtilT(const time_t input, Json::Value& output)
     {
-        tm timeInfo;
-#if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_XBOX)
-        gmtime_s(&timeInfo, &input);
-#elif defined(PLAYFAB_PLATFORM_LINUX) || defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_PLAYSTATION)
-        timeInfo = *gmtime(&input);
-#endif
-        char buff[40];
-        strftime(buff, 40, "%Y-%m-%dT%H:%M:%S.000Z", &timeInfo);
-        output = Json::Value(buff);
+        output = Json::Value(LocalTimeTToUtcString(input));
     }
-
     inline void FromJsonUtilT(const Json::Value& input, time_t& output)
     {
-        if (input == Json::Value::null)
-        {
-            return;
-        }
-
-        const std::string timeStr = input.asString();
-        tm timeStruct = {};
-        std::istringstream iss(timeStr);
-        iss >> std::get_time(&timeStruct, "%Y-%m-%dT%T");
-#if defined(PLAYFAB_PLATFORM_PLAYSTATION)
-        output = mktime(&timeStruct);
-#elif defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX)
-        output = timegm(&timeStruct);
-#else
-        output = _mkgmtime(&timeStruct);
-#endif
+        if (input == Json::Value::null) return;
```

Comment:
> return; [](start = 39, length = 8)

nit: I think I've seen deb specifically request braces on all if-else statements even if they are one line. if that's true you should fix this (it's certainly my preferred style)